contracts/modules/social_recovery/storage/GuardianStorage.sol

```diff
@@ -30,24 +30,22 @@ contract GuardianStorage is IGuardianStorage {
 
     /**
      * @dev Throws if the caller is not an enabled module.
-     * @param _wallet The target wallet.
      */
-    modifier onlyModule(address _wallet) {
+    modifier onlyModule() {
         // solhint-disable-next-line reason-string
-        require(ISafe(payable(_wallet)).isModuleEnabled(msg.sender), "GS: method only callable by an enabled module");
+        require(ISafe(payable(msg.sender)).isModuleEnabled(address(this)), "GS: method only callable by an enabled module");
```

Comment:
I think the revert message is wrong - it is only callable my an `ISafe`, and if the module is enabled. Is there a reason for this restriction? Shouldn't this just work as `authorized(_wallet)` from before?

Reply:
The revert message is indeed incorrect. Fixed here: https://github.com/5afe/CandideWalletContracts/pull/34/commits/81947fe68e75fa56973d87e046392f8da96def61.

This modifier is used to restrict any change in guardians list when module is disabled by the wallet

Reply:
Do we _want_ to restrict changes to the guardian list when the module is disabled?